apps/src/templates/instructions/ThreeColumns.jsx

```diff
@@ -57,4 +58,4 @@ ThreeColumns.propTypes = {
   }
 };
 
-export default ThreeColumns;
+export default Radium(ThreeColumns);
```

Comment:
You add Radium to this component but are not using (still using Object.assign for the root component style).  Missed step?